doc/nrf/config_and_build/config_and_build_system.rst

```diff
@@ -276,12 +276,49 @@ To disable these warnings, disable the :kconfig:option:`CONFIG_WARN_EXPERIMENTAL
 Sysbuild enabled by default
 ===========================
 
-When building :ref:`workspace applications <create_application_types_workspace>` copied from :ref:`nRF repositories <dm_repo_types>`, using the :ref:`standard building procedure <building>` automatically includes :ref:`configuration_system_overview_sysbuild` (the ``--sysbuild`` parameter).
-For this reason, unlike in Zephyr, ``--sysbuild`` does not have to be explicitly mentioned in the command prompt when building the application.
+The |NCS| `modifies the default behavior <sdk-zephyr west build patch_>`_ of ``west build``, so that the :ref:`standard building procedure <building>` will use sysbuild by default for :ref:`repository applications <create_application_types_repository>` in the :ref:`SDK repositories <dm_repo_types>`.
+For this reason, unlike in Zephyr, ``--sysbuild`` does not have to be explicitly mentioned in the command prompt when building a repository application.
 
-This rule does not apply if you work with out-of-tree :ref:`freestanding applications <create_application_types_freestanding>`, for which you need to manually pass ``--sysbuild`` to build commands in every case.
+This setting only applies to repositories delivered with the |NCS|, to maintain compatibility with child/parent images.
+It does not apply to out-of-tree applications, such as :ref:`freestanding applications <create_application_types_freestanding>`.
```

Comment:
Out-of-tree applications can be workspace or freestanding. It doesn't apply to either, so you need to rework this section a bit.